docs/UserManuals/ConfigUI/GitLab.md

```diff
@@ -49,3 +49,9 @@ There are no transformation rules for GitLab repos.
 
 ### Step 4 - Setting Sync Frequency
 You can choose how often you would like to sync your data in this step by selecting a sync frequency option or enter a cron code to specify your prefered schedule.
+
+
+## FAQ
+
+If you run into any problem, try finding the answer in [FAQ](../FAQ.md)
```

Comment:
If you run into any problems, please check the [FAQ](../FAQ.md) or create an [issue](https://github.com/apache/incubator-devlake/issues/new/choose).